src/algokit/core/bootstrap.py

```diff
@@ -9,6 +9,10 @@
 
 from algokit.core import proc, questionary_extensions
 
+import toml
+from algokit.core.version_prompt import get_latest_version_or_cached
```

Comment:
this function returns the latest version available on GitHub, for this check we want to check the current *executing* version, to do that use this 
`from algokit.core.conf import get_current_package_version`